switchbot_api/__init__.py

```diff
@@ -142,6 +142,15 @@ class LightCommands(Commands):
     BRIGHTNESS_DOWN = "brightnessDown"
 
 
+class VacuumCommands(Commands):
+    """Vacuum commands."""
+    
+    START = "start"
+    STOP = "stop"
+    DOCK = "dock"
+    FAN_SPEED = "PowLevel"
```

Comment:
I think we should try to be close to the vocabulary used by Switchbot, can we rename `FAN_SPEED` into `POWER_LEVEL` ?

Reply:
Yeah you're right, I was in HA mode and they call it fan speed.